src/client/js/formSentiment.js

```diff
@@ -0,0 +1,53 @@
+function handleSentiment(event) {
+    event.preventDefault();
+
+    // check what text was put into the form field
+    const txt = document.getElementById('text').value;
+    const language = document.getElementById('sentimentLanguage').value;
+    const resultContainer = document.querySelector('.sentiment.result');
+    const spinnerContainer = document.getElementById('spinner');
+
+    document.querySelectorAll('.result').forEach(it => {
+        it.style.display = 'none';
+    });
+    spinnerContainer.style.display = 'inherit';
+
+    console.log('::: Form Submitted :::');
+    fetch('http://localhost:8081/sentiment', {
```

Comment:
this should be done via an environment variable later, but for now it's ok. You won't be able to deploy your code (I mean, you would be, but it wouldn't work).

Reply:
I'll keep it in mind!